### PR TITLE
Bump gorm.io/driver/postgres from 1.3.10 to 1.4.5

### PR DESCRIPTION
**Pull request by carlory**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
related pr #428

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by carlory** [comment]
cc @Iceber

File: go.mod [hunk 1]
```diff
@@ -16,9 +16,9 @@ require (
 	go.uber.org/atomic v1.10.0
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gorm.io/datatypes v1.0.7
-	gorm.io/driver/mysql v1.4.3
-	gorm.io/driver/postgres v1.3.10
-	gorm.io/gorm v1.24.0
+	gorm.io/driver/mysql v1.3.6
```
Comment (Iceber) [inline go.mod:19]: We cannot downgrade the mysql version.

File: go.mod [hunk 2]
```diff
@@ -16,9 +16,9 @@ require (
...
+	gorm.io/driver/postgres v1.4.5
+	gorm.io/gorm v1.24.1-0.20221019064659-5dd2bb482755
```
Comment (Iceber) [inline go.mod:21]: gorm 1.24 has been released, we can upgrade gorm to a stable version first.
https://github.com/clusterpedia-io/clusterpedia/pull/441

**Pull request merged**